v23.1/tsquery.md

```diff
@@ -0,0 +1,63 @@
+---
+title: TSQUERY
+summary: The TSQUERY data type a list of lexemes separated by operators, and is used in full-text search.
+toc: true
+docs_area: reference.sql
+---
+
+The `TSQUERY` [data type](data-types.html) stores a list of lexemes separated by operators. `TSQUERY` values are used in [full-text search](full-text-search.html).
+
+## Syntax
+
+A `TSQUERY` comprises individual lexemes and operators in the form: `'These' & 'lexemes' & 'are' & 'not' & 'normalized.'`.
+
+The operators in a `TSQUERY` are used to [match a `TSQUERY` to a `TSVECTOR`](full-text-search.html#match-queries-to-documents). Valid `TSQUERY` operators are:
+
+- `&` (AND)
+- `|` (OR)
+- `!` (NOT)
+- `<->` (FOLLOWED BY)
+
+You can optionally add the following to each lexeme:
+
+- One or more weight letters (`A`, `B`, `C`, or `D`): 
+
+	`'These' & 'lexemes':B & 'are' & 'not' & 'normalized.':A`
+
+	If not specified, a lexeme's weight defaults to `D`. For more information about weights, see the [PostgreSQL documentation](https://www.postgresql.org/docs/15/datatype-textsearch.html#DATATYPE-TSQUERY).
+
+To be usable in [full-text search](full-text-search.html), the lexemes **must be normalized**. You can do this by using the `to_tsquery()`, `plainto_tsquery()`, or `phraseto_tsquery()` [built-in functions](functions-and-operators.html#full-text-search-functions) to convert a string input to `TSQUERY`:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT to_tsquery('These & lexemes & are & not & normalized.');
+~~~
+
+~~~
+      to_tsquery
+----------------------
+  'lexem' & 'normal'
+~~~
+
+Normalization removes the following from the input:
+
+- Suffixes.
```

Comment:
We could add more detail to this - it kind of looks like suffixes, but really each of the lexemes is normalized according to an algorithm called "stemming" (the specific implementation is "snowball stemming"). It's true that it does look like removal of suffixes but defining suffixes isn't super easy.